app/api/ada/adaTransactions/adaTransactionsHistory.js

```diff
@@ -42,6 +43,9 @@ import {
   getLastBlockNumber,
   getSingleCryptoAccount
 } from '../adaLocalStorage';
+import type {
+  CryptoAccount,
+} from '../adaLocalStorage';
```

Comment:
Why two new imports are added and they are the only changes in the file?

Reply:
Because `CryptoAccount` used to be a globally declared type from the old bindings. Since the old bindings don't exist anymore (but we still need this type because this is what we serialize for `localstorage` I had to add it back in and import it like you would any other type.

Old type definition: https://github.com/Emurgo/yoroi-frontend/blob/develop/flow/declarations/CardanoCrypto.js#L187